exdb/views.py

```diff
@@ -51,9 +51,11 @@ def get_hs_queryset(self):
         experience_approvals = ExperienceApproval.objects.filter(
             approver=self.request.user, experience__status='ad'
         )
+        next_approver_queue = Q(next_approver=self.request.user) & ~Q(status="dr")
```

Comment:
That should work.  We do need to filter out only pending experiences in the next_approver queue in whatever the final query is.  Else hs can see drafts (or any other experience) where they are set as the next approver.
I can work on it or you, your choice.